tests/Unit/Infrastructure/BestEffortDonationEventLoggerTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace Unit\Infrastructure;
```

Comment:
my IDE hints that it wants a different namespace:
```suggestion
namespace WMDE\Fundraising\DonationContext\Tests\Unit\Infrastructure;
```